### ISSUE TITLE
question about melonDS

### ISSUE THREAD

**Issue by Gygrfynhvfcddeft** [BinBashBanana/webretro#111]
how does the save work? like i tried but it didn't work

**Comment by BinBashBanana** [comment]
that's really helpful

**Comment by RuralAnemone** [comment]
what did you try? in-game saves have to also be written to your browser's storage: the emulator autosaves every 5 minutes and you can manually save from the menu.

**Comment by Gygrfynhvfcddeft** [comment]
No, I saved in game and saved manually from the menu bar and when I exited the page it just didn't work when I started it up again.

**Comment by BinBashBanana** [comment]
The game likely did not save before the emulator's save was written. Be sure to use the in-game save and then use the "save game" button
